src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -87,13 +73,15 @@ const TaskTable = ({ project, loggedInPersonId, filter }: Props) => {
     if (project && open) {
       getTasksAndWorkHours();
     }
-  }, [project, open, filter]);
+  }, [project, open]);
 
   useEffect(() => {
-    setPhase([]);
-    setOpen(false);
-  }, [loggedInPersonId]);
-
+    if (loggedInPersonId) {
+      setPhase([]);
+      setOpen(false);
+    }
+  }, []);
```

Comment:
Can't this just be removed, on component render you are setting these values to the same values you are initializing the state as- so why is this needed?